kolibri/plugins/coach_tools/assets/src/vue/reports/summary-section/index.vue

```diff
@@ -145,7 +177,24 @@
   @require '~kolibri.styles.coreTheme'
 
   .summary-section
-    padding: 20px
+    display: table
+    width: 100%
     background-color: white
 
+  .summary-section-row
+    display: table-row
+
+  .summary-section-details,
+  .summary-section-progress,
+  .summary-section-date
+    display: table-cell
+    padding: 10px
+    vertical-align: top
+
+  .summary-section-progress
+    width: 20%
+
+  .summary-section-date
+    width: 150px
```

Comment:
I was trying to require the file, but couldn't figure it out. How would I require that local stylesheet?